src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTO.kt

```diff
@@ -27,7 +27,7 @@ data class DraftReferralDTO(
   val additionalRiskInformation: String? = null,
   val usingRarDays: Boolean? = null,
   val maximumRarDays: Int? = null,
-  val desiredOutcomesIds: List<UUID>? = null,
+  val desiredOutcomes: Map<UUID, List<UUID>>? = null,
```

Comment:
Realise that I should be using a new DTO object rather than Map based on the code in the UI.
Going to make that change quickly just now...